recipe/meta.yaml

```
@@ -79,19 +68,30 @@ test:
     - skbio.tree.tests
     - skbio.util
     - skbio.util.tests
+  requires:
+    - pip
+    - pytest
+    - responses
+    - matplotlib
+    - biom-format
+    - array-api-compat
+  script_env:
+    - MPLBACKEND=Agg
   commands:
-    - MPLBACKEND='Agg' python -m skbio.test
+    - pip check
+    - python -c "from importlib.metadata import version as v; assert v('scikit-bio')=='{{ version }}'"
+    - python -m skbio.test
 
 about:
-  home: http://scikit-bio.org
+  home: https://scikit-bio.org
   license: BSD-3-Clause
   license_family: BSD
-  license_file: COPYING.txt
-  summary: 'Data structures, algorithms and educational resources for bioinformatics.'
+  license_file: LICENSE.txt
+  summary: Data structures, algorithms and educational resources for bioinformatics.
   description: |
     scikit-bio is an open-source, BSD-licensed, python package providing data
     structures, algorithms, and educational resources for bioinformatics.
-  doc_url: http://scikit-bio.org
+  doc_url: https://scikit-bio.org
   doc_source_url: https://github.com/biocore/scikit-bio/blob/master/doc/source/index.rst
   dev_url: https://github.com/biocore/scikit-bio
```

And https://github.com/scikit-bio/scikit-bio/blob/0.7.0/pyproject.toml#L53

```suggestion
  dev_url: https://github.com/scikit-bio/scikit-bio
```